code/go/0chain.net/smartcontract/zcnsc/context_helpers_test.go

```diff
@@ -239,6 +239,192 @@ func MakeMockStateContext() *mockStateContext {
 	return ctx
 }
 
+func MakeMockStateContextWithNoAutorizers() *mockStateContext {
```

Comment:
@peterlimg Since golang neither supports optional parameters nor method overloading so I was finding it difficult to add modify MakeMockStateContext function to conditional add or not add the authorizers. So I copied the method to a new method MakeMockStateContextWithNoAutorizers and removed the part responsible for adding autorizers.

Can you suggest a better way of achieving the same thing?